apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.ts

```diff
@@ -26,7 +26,7 @@ const sql = sqldb.loadSqlEquiv(import.meta.url);
 
 router.get(
   '/',
-  asyncHandler(async (req, res) => {
+  typedAsyncHandler<'course', { needToSync: boolean }>(async (req, res) => {
```

Comment:
Nit: consider moving this out of `res.locals` (maybe in another PR first). The fact that this is attached there and not passed directly to the HTML template si probably just a relic of the EJS days.